cookiecutter.json

```diff
@@ -4,6 +4,7 @@
   "github_username": "yourname",
   "hack_oregon_team": "Transportation Systems",
   "year": "2019",
+  "project_root_path": "transportation",
```

Comment:
Is this value intentionally different than `project_root_path` from the README?  If so, can you clarify in the README how to manage the two?

Reply:
@MikeTheCanuck realized this value probably should be templatized based on the 'hack_oregon_team' variable, so will have to play around with it, but I think basic logic should be:
* 1st word of team name
* lower-case

But will really be whatever path we point the load balancer to. Transportation is unique I guess because of the "transportation2019" route path, which I don't think we plan on as standard?

So plan to correct here:

1. update readme to match  the "transportation" path
2. templatize the json file to autogenerate value based on business logic (first word, lowercase)

Reply:
I'm not sure which way you're leaning, so here's the current situation:
- 2017 Transportation API occupies service.civicpdx.org/transport
- 2018 Transportation API occupies service.civicpdx.org/transportation-systems/
- we'd proposed `transportation2019` as the standard name for this year's project, which would suggest service.civicpdx.org/transportation2019/ as the base route for the APIs

If you think service.civicpdx.org/transportation/ is the better choice, we can update the CloudFormation config - but as an API user, I could get confused between /transport/ and /transportation/, which is why I thought we were being less ambiguous.

Reply:
so, not sure if we need to exactly match the actual transportation project here, more of show off what the general pattern would be, so within the standardized template I standardizing to `/transportation`, but we can keep `/transportation2019` as the actual value in cloud formation.
Created pr here off this branch to resolve:
https://github.com/hackoregon/2019-backend-cookiecutter-django/pull/41